File: alyak-api-server/src/main/java/com/github/seungjae97/alyak/alyakapiserver/domain/schedule/dto/response/ScheduleBackupResponse.java
package com.github.seungjae97.alyak.alyakapiserver.domain.schedule.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.LocalTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ScheduleBackupResponse {

    private Long scheduleId;
    private Long pillId;
    private String pillName;
    private Integer dosage;
    private LocalTime scheduledTime;
    private LocalDate startDate;
    private LocalDate endDate;
    private LocalDateTime createdAt;
}

File: application_project/app/src/main/java/com/alyak/detector/feature/notification/data/model/ScheduleBackupResponse.kt
package com.alyak.detector.feature.notification.data.model

import com.google.gson.annotations.SerializedName

data class ScheduleBackupResponse(
    @SerializedName("scheduleId") val scheduleId: Long?,
    @SerializedName("pillId") val pillId: Long?,
    @SerializedName("pillName") val pillName: String?,
    @SerializedName("dosage") val dosage: Int?,
    @SerializedName("scheduledTime") val scheduledTime: String?,
    @SerializedName("startDate") val startDate: String?,
    @SerializedName("endDate") val endDate: String?,
    @SerializedName("createdAt") val createdAt: String?,
)
